app/models/customerInformation/CustomerInformation.scala

```diff
@@ -67,6 +76,8 @@ object CustomerInformation {
     contactPreferencePath.readNullable[String].orElse(Reads.pure(None)) and
     changeIndicatorsPath.readNullable[ChangeIndicators].orElse(Reads.pure(None)) and
     nameIsReadOnlyPath.readNullable[Boolean].orElse(Reads.pure(None)) and
-    partyTypePath.readNullable[String].orElse(Reads.pure(None))
+    partyTypePath.readNullable[String].orElse(Reads.pure(None)) and
+    isInsolventPath.read[Boolean].orElse(Reads.pure(false)) and
```

Comment:
Could you remove the `orElse` bit here for the `isInsolvent` field as this should always be returned by the API. We would the exception to be thrown if they don't return this field to us rather than assuming it's false.